[DOCS] Host Isolation Exceptions - IP Allowlist (#1190)
elastic/security-docs: docs/management/admin/host-isolation-exceptions.asciidoc

[[host-isolation-exceptions]]
[chapter]
= Host isolation exceptions

> You can configure host isolation exceptions for specific IP addresses that <<host-isolation-ov, isolated hosts>> are still allowed to communicate with after they've been blocked from the rest of your network. Isolated hosts can still send data to {es} and {kib}, so you don't need to set up host isolation exceptions for them.

Author: ```suggestion
You can configure host isolation exceptions for specific IP addresses that <<host-isolation-ov, isolated hosts>> are still allowed to communicate with, even when blocked from the rest of your network. Isolated hosts can still send data to {es} and {kib}, so you don't need to set up host isolation exceptions for them.
```

> Host isolation exceptions support IPv4 addresses, with optional CIDR (classless inter-domain routing) notation.

Reviewer: ```suggestion
Host isolation exceptions support IPv4 addresses, with optional classless inter-domain routing (CIDR) notation.
```

Reviewer: Recommending this switch because I've generally seen the definition first and the acronym second.

> IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location, since you’re allowing it to communicate with hosts that you've isolated due to suspicious activity.

Reviewer 2: Another point to make here is that users isolate hosts to stop the potential spread of a threat.  So any IP they except could still be at risk.

You do mention that it's a `secure location`, so maybe that's enough.  Was just wondering if there was a more up front way to make that risk of a threat spreading more clear.

Author: Good point! How about something like this -- does this go far enough to make the risk clear?
```suggestion
IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location, since you’re allowing it to communicate with hosts that you've isolated to contain a potential threat and prevent it from spreading.
```

Reviewer 2: looks good!

> IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location, since you’re allowing it to communicate with hosts that you've isolated to contain a potential threat and prevent it from spreading.

Reviewer: ```suggestion
IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location since you’re allowing it to communicate with hosts that you've isolated to contain a potential threat and prevent it from spreading.
```

Reviewer: Can the second part of this sentence be shortened at all? Maybe to something like "since you're allowing it to communicate with potentially dangerous hosts that you've isolated."

Author: @nastasha-solomon Yeah, it is a long sentence. Earlier, @kevinlog and I revised this specifically to convey the idea of limiting a spreading threat, so instead of omitting that info, maybe tighten up the phrasing?
```suggestion
IMPORTANT: Each host isolation exception IP address should be a highly trusted and secure location since you’re allowing it to communicate with hosts that you've isolated to prevent a potential threat from spreading.
```

Reviewer: Your revision looks good! The only tweak I'd suggested is to change "you've isolated" to "are isolated" or "have been isolated" because anyone could've isolated the host.

Author: Good point; updated and committed above!

. Fill in these fields in the **Add Host isolation exception** flyout:
.. `Name your host isolation exceptions`: Enter a name to identify the host isolation exception.
.. `Description`: Enter a description to provide more information on the host isolation exception (optional).
.. `Enter IP Address`: Enter the IP address that you want an isolated host to be able to communicate with. This must be an IPv4 address, with optional CIDR notation (for example, `0.0.0.0` or `1.0.0.0/24` respectively).
. Click **Add Host isolation exception**.

[discrete]
[[manage-host-isolation-exceptions]]
== View and manage host isolation exceptions

The **Host isolation exceptions** list displays all the host isolation exceptions that have been configured for {elastic-sec}.

[role="screenshot"]
image::images/host-isolation-exceptions-ui.png[List of host isolation exceptions]

To refine the Host isolation exceptions list by a specific value, enter a query in the search bar. You can search by name, description, or IP address.

To edit a host isolation exception:

. Click the actions button (**...**) for the exception you want to edit, then select **Edit Exception**.

> . Modify details or conditions as needed.

Author: ```suggestion
. Modify details as needed.
```

. Click **Edit Host isolation exception**. The newly modified exception appears at the top of the list.

To delete a host isolation exception:

. Click the actions button (**...**) for the exception you want to delete, then select **Delete Exception**.

> . On the dialog that opens, verify that you are removing the correct host isolation exception, then click **Remove exception**. A confirmation message is displayed.

Reviewer: I'm not quite sure whether `on` is the appropriate preposition to use here, or if it should be `from`. I suppose both of them work. WDTY?

Author: I'd say "on" (or maybe "in") for this case, since the user is looking at information _on_ or _in_ a dialog box. IMO "from" would be more appropriate if they were doing something active like dragging an item _from_ one location to another, or maybe selecting an option _from_ a list or drop-down.
